src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -413,13 +413,13 @@ export function openFacilityPageV2(page, uiSchema, schema) {
           newAppointment.eligibility[`${facilityId}_${typeOfCareId}`] || null;
 
         if (eligibilityDataNeeded && !eligibilityChecks) {
-          const selectedFacility = typeOfCareFacilities[0];
```

Comment:
If a user came back to this page, we were incorrectly checking eligibility for ` typeOfCareFacilities[0]` instead of the selected facility.